.travis.yml

```diff
@@ -4,6 +4,7 @@ notifications:
 rvm:
   - 2.2.4 # deployed
   - 2.3.4
```

Comment:
what about replacing the `2.3.4` entry?  or do you anticipate possibly running under both?

Reply:
I agree. There's no sense in having multiple 2.3 releases tested.

Reply:
Technically, it is running under both -- right now in 2.3.7 in test, and 2.3.4 in prod. My thought was to remove whatever entry once we have this running in test and prod at the same version.